sites/website/build-homepage.js

```diff
@@ -0,0 +1,39 @@
+// FIXME This file is a temporary script and should be obsoleted asap
+
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/typedef */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { spawn } = require("child_process");
+const path = require("path");
+const copyfiles = require("copyfiles");
+
+const projectPath = path.resolve(__dirname, "../fast-website");
+const staticPath = path.resolve(__dirname, "static");
+const indexPath = path.resolve(projectPath, "dist/index.html");
+
+// Run `yarn build` in the fast-website project
+const yarnBuild = spawn("yarn", ["build"], { cwd: projectPath });
+
+const copyfilesConfig = { flat: true, verbose: true, up: true };
```

Comment:
do you think it may be more readable to organize the `yarnBuild` routines in a single group and put `copyfilesConfig` just above?